communications-handbook.md

```diff
@@ -0,0 +1,155 @@
+
+Purpose:
+
+* Disseminate official information about the association
+* Conduct the business of the association
+* Host the main organizational website 
+* Host information related to association events and activities
+* Organizational email (run through a GSuite account)
+
+### Website
+
+Hosted via a GitHub repo and site: https://github.com/USRSE/usrse.github.io; #website channel on Slack
+
+Guidelines, procedures, etc. for the managing and updating the website are determined by the website committee and documented in the GitHub repo. The current guidelines for incremental changes require the use of a PR and a review of the PR. The guidelines are documented here  https://github.com/USRSE/usrse.github.io/blob/master/.github/CONTRIBUTING.md.  
+
+Large changes to the website including the creation of a new site, platform, subdomain, website section, etc. or a major reorganization or change in theme, styling, function of existing content must be first 1) discussed and agreed to by the Website Committee, and then 2) brought to the Steering Committee, before being added to the domain to ensure they are aligned with the mission of the Association. This includes changes to existing sites that significantly alter the way they function or content they provide. New sites or major changes to functionality should not be deployed under the us-rse.org domain before being reviewed. Instead, they should be previewed under a personal account or other resource. Contact the website committee for technical guidance. 
+
+### Email
+
+Distribution lists/groups: 
+
+* All US-RSE members are added to the announce@us-rse.org email list.  This is the list to be used for all important/official communications to the community.  Members of the Steering Committee can send to the list, but communications should generally come from the Steering Committee generally instead of individual members.
+* The contact@us-rse.org email is the main address for people to contact US-RSE leadership.  It is a google group that includes all of the Steering Committee members.
+* Discussion email lists can be created under the GSuite account, but members should opt into such lists, not be subscribed automatically.  
+* US-RSE committee leaders, working group chairs, etc. can request discussion or distribution email groups as needed for US-RSE activities.
+
+Individual accounts: We generally do not have personal @us-rse.org accounts/email addresses.  While a few do exist, they are for administering the GSuite account and related resources.  Instead of personal accounts, use/create positional or group accounts.  Currently, the main account to use to send email from (if needed) is leadership@us-rse.org.  See the Steering Committee Metadoc for login information.
+
+
+## Organizational GitHub Account
+
+https://github.com/usrse
+
+Purpose: 
+
+* Host the US-RSE website
+* Host US-RSE documents and artefacts as needed/appropriate
```

Comment:
Artifact?

Reply:
yes, technically both are correct, but I'm fine changing it